### PR TITLE
Adding to the French translation the missing files

### PR DESCRIPTION
**Pull request by 5axes**
Translations for "Enable Print Cooling" and "Raft Fan Speed".

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
This adds a bunch of images that are not used by any article, and don't seem to show anything in particular. Just a cropped and zoomed segment of some of the existing images?

**Comment by 5axes** [comment]
They are normaly used in skin_edge_support_thickness.md  On my point of view they are more clear than the default image. And they are also used in the French translation of the Cura Manual we offer on our FB group

**Comment by 5axes** [comment]
[Cura45.pdf](https://github.com/Ghostkeeper/SettingsGuide/files/4374291/Cura45.pdf)

File: resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-* Appliquez du papier de verre sur la plaque de montage pour la rendre plus rugueuse et augmenter la surface de contact. Ne la rendez pas trop rugueuse, sinon vous aurez une surface inégale pour imprimer. Pour ce faire, utilisez un papier de verre à grain très fin, d'un grain de 500 ou plus.
-* Des "feuilles d'adhérence" spécialisées sont vendues par de nombreux vendeurs d'imprimantes 3D, qui peuvent être tendues sur la plaque de construction. Elles fonctionnent de la même manière que le ruban adhésif, mais sont conçues pour durer plus longtemps. Certaines sont faites d'un plastique qui adhère chimiquement au matériau imprimé également.
+* La laque pour cheveux fonctionne de la même manière que la colle, en ce sens qu'elle laisse une surface rugueuse à grain fin sur le plateau. Il est très facile de l'appliquer uniformément.
+* Appliquez du papier de verre sur l plateau pour la rendre plus rugueuse et augmenter la surface de contact. Ne la rendez pas trop rugueuse, sinon vous aurez une surface inégale pour imprimer. Pour ce faire, utilisez un papier de verre à grain très fin, d'un grain de 500 ou plus.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md]: `le plateau?`

**Comment by Ghostkeeper** [comment]
Sorry, but I disagree. The translation should be a translation, not a separate guide. Also one of the reasons why I took this guide is because Ultimaker's online guide is in many ways incomplete, sometimes more towards marketing-talk than technical talk, and in some cases factually incorrect.

If the default images aren't clear, it's better to replace the originals so that it's better for everyone, not just the French public. I've written down guidelines for creating screenshots here: https://github.com/Ghostkeeper/SettingsGuide/blob/master/contributing.md#writing-articles These are important, as the Marketplace doesn't take plug-ins of more than about 30MB. At this point we have about 5MB left and there's still a lot to write about. I'm doing my best to make it look good while staying light.

**Comment by 5axes** [comment]
Ok

File: resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md (hunk 1 above)
Comment (5axes) [reply resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md]: Yes , "le plateau"  but should be "Utilisez du papier de verre sur le plateau pour le rendre plus rugueux"   Well I need to change all this section.

**Comment by 5axes** [comment]
Translation of the Machine settings.  + New svg file translation

**Comment by Ghostkeeper** [comment]
I'd like to pull this in, especially with all of the improvements and fixes you've made recently and now that the new articles are also properly translated. I can't really speak for the accuracy since I'm not a native, but lots of tweaks seems like a sign that it's had some good thought.

But since you were still making changes, I'll ask, are you done with the changes you wanted to make or is this still WIP?

There are still those images in there that I requested to remove. If necessary I can remove those myself as well when I merge it in.

**Comment by 5axes** [comment]
Concerning the machine parameter I think that the translataion is quite correct and you can add it to the regular release.  For the image yes you can remove it if you want .

**Comment by 5axes** [comment]
Every Links in the french translation are now correct

Best regards,


Laurent

**Comment by 5axes** [comment]
Every links in the french translation are now correct

**Comment by Ghostkeeper** [comment]
All right then. Good work all in all! I'll merge this in and remove those two unused images still.

**Pull request merged**